apps/sms/views/conversation/test/unit/conversation_test.js

```diff
@@ -5252,10 +5248,39 @@ suite('conversation.js >', function() {
       this.sinon.stub(Navigation, 'isCurrentPanel').returns(false);
       Navigation.isCurrentPanel.withArgs('composer').returns(true);
 
+      this.sinon.stub(Utils, 'confirm').returns(Promise.resolve());
+
       this.sinon.spy(Navigation, 'toPanel');
       ConversationView.initRecipients();
     });
 
+    test('send empty SMS to one Recipient', function() {
+      var recipient = '999';
+
+      ConversationView.recipients.add({
+        number: recipient
+      });
+
+      Compose.getContent.returns([]); //return empty message
+
+      clickButtonAndSelectSim();
```

Comment:
Also please add one more test for the case when user __doesn't__ confirm sending of empty message